src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import {Header} from './component'
 import './App.css'
+import SignUpForm from './component/SignUpForm/SignUpForm'
```

Comment:
Dentro de la carpeta `src/component/SignUpForm`, pueden crear un archivo `index.js` que exporte el componete SignUpForm (`export { SignUpForm } from './SignUpForm'`)  asi pueden importarlo de esta manera acá:
```suggestion
import SignUpForm from './component/SignUpForm'
```